runtime/src/main/kotlin/kotlin/native/Text.kt

```diff
@@ -71,11 +122,11 @@ public fun String.toUtf8() : ByteArray {
 @Deprecated("Use encodeToByteArray instead", ReplaceWith("encodeToByteArray(start, start + size)"))
 public fun String.toUtf8(start: Int = 0, size: Int = this.length) : ByteArray {
     checkBoundsIndexes(start, start + size, this.length)
-    return toUtf8Impl(start, size)
+    return unsafeToUtf8Impl(start, size)
 }
 
-@SymbolName("Kotlin_String_toUtf8")
-internal external fun String.toUtf8Impl(start: Int, size: Int) : ByteArray
+@SymbolName("Kotlin_String_unsafeToUtf8")
+internal external fun String.unsafeToUtf8Impl(start: Int, size: Int) : ByteArray
```

Comment:
Since we're renaming it anyway, I propose to name it `unsafeStringToUtf8Impl` to be consistent with `unsafeStringFromUtf8Impl`

Reply:
Renamed to `unsafeStringFromUtf8